packages/react-recommendations/src/ListView.tsx

```diff
@@ -24,7 +24,11 @@ export function ListView<TItem extends RecordWithObjectID>(
             key={item.objectID}
             className={cx('auc-Recommendations-item', props.classNames.item)}
           >
-            <props.itemComponent item={item} />
+            <props.itemComponent
+              createElement={createElement}
+              Fragment={Fragment}
```

Comment:
The same comment as above applies here.

Otherwise, if we change the `view` prop, users won't have access to `createElement` and `Fragment`.

Reply:
See https://github.com/algolia/ui-components/pull/26#discussion_r642932136